shared/src/main/java/uk/gov/di/authentication/shared/services/AuditService.java

```diff
@@ -39,7 +39,7 @@ public AuditService(ConfigurationService configurationService) {
 
     public void submitAuditEvent(
             AuditableEvent event,
-            String requestId,
+            String clientSessionId,
```

Comment:
Were we not using the requestId?
Are txma expecting the clientSessionId to come through like this?

Reply:
Yes the request Id was redundant. This is now replaced with client session id and captured in Txma's `withGovukSigninJourneyId`. I will need to do a follow up with Txma to ensure that's pulling through.